CHANGELOG.rst

```diff
@@ -8,8 +8,8 @@ CHANGELOG
 * doc-fix: update information about saving models in the MXNet README
 * doc-fix: change ReadTheDocs links from latest to stable
 * doc-fix: add ``transform_fn`` information and fix ``input_fn`` signature in the MXNet README
-* feature: Support for ``Predictor`` to delete endpoint configuration by default when calling ``delete_endpoint()``
-* feature: Support for ``model`` to delete SageMaker model
+* feature: Support for ``Predictor`` to delete model, and delete endpoint configuration by default when calling ``delete_endpoint()``
```

Comment:
Your feature addition needs to be separate and in a new version, as your last PR go merged and released to PyPI already.

https://github.com/aws/sagemaker-python-sdk/releases/tag/v1.18.3

Reply:
ohh. So the next version should be 1.18.4.dev?

Reply:
I would say 'add support for..." to be consistent

Reply:
even better would be something like "feature: Predictor: delete model and endpoint configuration by default when calling `delete_endpoint()`", though we've been extremely inconsistent in keeping that format